cypress/e2e/auth_spec/redirect.cy.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+
+import { cy, describe, it } from "local-cypress";
+
+describe("redirect", () => {
+  it("Check if login redirects to the right url", () => {
+    cy.log("Logging in the user staffdev:Coronasafe@123");
+
+    cy.visit(
+      "/facility/657c32be-d584-476c-9ce2-0412f0e7692e/cns?page=1&limit=6"
+    );
```

Comment:
Avoid hard-coding the UUID into tests. This entity may not be present in the cypress test environment.

Consider clicking and opening the first facility page instead.

Reply:
Clicking on the facility page won't work since there isn't a login yet but yes i understand the concern will use an existing endpoint like `/resource/board` instead.